froide/foirequest/forms/request.py

```diff
@@ -449,3 +450,43 @@ def save(self) -> List[str]:
             m for m in trigger.apply_actions(self.foirequest, self.request) if m
         ]
         return messages
+
+
+class RedactDescriptionForm(forms.Form):
+    description = forms.CharField(required=False)
+    description_length = forms.IntegerField()
+
+    def clean_description(self):
+        val = self.cleaned_data["description"]
+        if not val:
+            return []
+        try:
+            val = [int(x) for x in val.split(",")]
+        except ValueError:
+            raise forms.ValidationError("Bad value")
+        return val
+
+    def save(self, request):
```

Comment:
This is a `FoiRequest` and should be named `foirequest` to avoid confusion with `HttpRequest`